src/ui/optionsdialog.cc

```diff
@@ -18,6 +18,7 @@
 
 #include <QMessageBox>
 
+#include <ui/hexedit.h>
```

Comment:
Please fix:
* Include grouping
* Use of `<>`

Reply:
Done.